src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
The Rectangle class should not be an empty class. It's better to add a constructor that initializes the length and width fields instead of using setters after object creation. This ensures that you always have a fully initialized object.